Formula/argocd.rb

```diff
@@ -0,0 +1,27 @@
+class Argocd < Formula
+  desc "GitOps Continuous Delivery for Kubernetes"
+  homepage "https://argoproj.io"
+  url "https://github.com/argoproj/argo-cd.git",
+      :tag      => "v1.6.1",
+      :revision => "159674ee844a378fb98fe297006bf7b83a6e32d2"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    inreplace "Makefile", "CGO_ENABLED=0", ""
```

Comment:
Why this change?

Reply:
Same problem as with the `argo` formula. It fails with these errors:
```
go build os/user: copying /private/tmp/go-build946619233/b142/pkg.a: open /usr/local/Cellar/go/1.14.5/libexec/pkg/darwin_amd64/os/user.a: operation not permitted
go build net: copying /private/tmp/go-build946619233/b074/pkg.a: open /usr/local/Cellar/go/1.14.5/libexec/pkg/darwin_amd64/net.a: operation not permitted
```

Reply:
might be good to have this as comment for future reference.

Reply:
@chenrui333 I added a comment